packages/playwright/src/transform/transform.ts

```diff
@@ -92,14 +92,14 @@ function loadAndValidateTsconfigsForFile(file: string): ParsedTsConfigData[] {
 const pathSeparator = process.platform === 'win32' ? ';' : ':';
 const builtins = new Set(Module.builtinModules);
 
-export function resolveHook(filename: string, specifier: string): string | undefined {
+export function resolveHook(filename: string, specifier: string, dontResolveDirectories = false): string | undefined {
```

Comment:
Let's call the new argument `isModule` and make it required. This will expose all the places where we need to think about it.

Reply:
Could you explain why you'd call it `isModule`? Because of "module loader"?

The other callsites of `resolveHook` are inside component testing, and are used to turn Vite's import info into filenames. I think the changes we're making here also apply there, so I don't think we even need a parameter. I've removed it in 8fe370acf8e4e802f00e640c7ba0eb62069bf5e4.